sync_ci/pkg/parser/regex_rules_test.go

```diff
@@ -0,0 +1,156 @@
+package parser
+
+import (
+	"bufio"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestRegex_TestRules(t *testing.T) {
+	updateRegexpRules("./pkg/parser/regex_rules.json")
```

Comment:
panic: open ./pkg/parser/regex_rules.json: no such file or directory [recovered]

Reply:
please change pwd to ./pkg